benchmarks/fasta/ruby/bench.rb

```diff
@@ -62,6 +63,7 @@ def make_random_fasta(table, n)
     prob = 0.0
     rwidth = (1..width)
     table.each{|v| v[1]= (prob += v[1])}
+    table = table.collect{|v| prob += v[1]; [v[0], prob] }
```

Comment:
I don't know if I'm missing something here:

 * line 65 mutates table (from function scope) in-place and mutates prob (also in function scope).
 * line 66 mutates the same prob, and creates a new table.

So prob is mutated twice where before it was mutated once. Did you mean to to remove line 65?